handling/routes.go

```diff
@@ -0,0 +1,135 @@
+package handling
+
+import (
+	"chmgt/models"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/justinas/alice"
+)
+
+type Env struct {
+	DB models.Datastore
+}
+
+// Route provides all of the items needed to build a route
+type Route struct {
+	Name        string
+	Method      string
+	Pattern     string
+	Description string
+	Handler     http.Handler
+}
+
+// RouteDefs provides an array of routes
+var RouteDefs []Route
+
+// NewRouter builds the routing structure
+func NewRouter() *mux.Router {
+	db, err := models.NewDB("chmgt.db")
```

Comment:
It appears my comment was removed. I think I hit a wrong button, d'oh!

Anyhoo, I think this line should have `models.DSN` as the arg to `models.NewDB` instead of the hardcoded file name.